### PR TITLE
Update Persian News Service Podcast Promo Config

### LINKED ISSUE #9466: Update Persian podcast promo
**Is your feature request related to a problem? Please describe.**
Update the Persian podcast promo to show a more up to date brand. 

**Describe the solution you'd like**

New config as follows

Service | Podcast Pid | title: | brandTitle: | brandDescription: | src: | alt: | text: | href: | Title validated | pop/unpop/single
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Persian | p036kbdd | پادکست | به عبارت دیگر | گفتگو با چهارهای سیاسی، هنری و ادبی از تلویزیون فارسی بی بی سی. | http://ichef.bbci.co.uk/images/ic/3000x3000/p036kbm2.jpg | پادکست | برنامه ها | https://www.bbc.com/persian/podcasts/p036kbdd |   | n | unpop

Update the Persian config found here - https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/persian.js

**Describe alternatives you've considered**

**Testing notes**

Validate image & content looks OK on Test

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves https://github.com/bbc/simorgh/issues/9466

**Overall change:**
Updates Persian service config file to reflect new podcast promo

**Code changes:**

- Updates config files for Persian service
- Adds an article and podcast promo for local testing
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/persian.js [hunk 1]
```diff
@@ -66,15 +66,16 @@ export const service = {
...
       title: 'پادکست',
-      brandTitle: 'داستان انقلاب به روایت بی‌ بی سی',
-      brandDescription: 'مجموعه داستان انقلاب از آرشیو رادیوی بی بی سی',
+      brandTitle: 'به عبارت دیگر',
+      brandDescription:
+        'گفتگو با چهارهای سیاسی، هنری و ادبی از تلویزیون فارسی بی بی سی.',
       image: {
-        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p0703j1b.jpg',
-        alt: 'داستان انقلاب به روایت بی‌ بی سی',
+        src: 'http://ichef.bbci.co.uk/images/ic/$recipe/p036kbm2.jpg',
```
Comment (jroebu14) [inline src/app/lib/config/services/persian.js]: should this be `https`?

Comment (ryanmccombe) [reply src/app/lib/config/services/persian.js]:
```suggestion
        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p036kbm2.jpg',
```

**Review by ryanmccombe — APPROVED** [review]
👍

File: src/app/lib/config/services/persian.js (hunk 1 above)
Comment (pvaliani) [reply src/app/lib/config/services/persian.js]: Ye.  good spot - when we receive links from Editorial they come in without. We have had to update recent configs with that recently. Will get it updated. Thanks.

Comment (pvaliani) [reply src/app/lib/config/services/persian.js]: done!

**Review by MuhammadAHussain — APPROVED** [review]
LGTM 👍

**Comment by skwong25** [comment]
### Testing

Run locally cross-browser & cross-device: 
- Desktop: Firefox, Chrome, Safari, Opera
- Mobile: iOS (Chrome, Opera Mini), Android (Chrome, Firefox)  

All looks fine except one query re. hover state for the bottom-most link in Desktop Firefox - underline seems visually unclear: 

Firefox

**Comment by pvaliani** [comment]
> ### Testing
> Run locally cross-browser & cross-device:
> 
> * Desktop: Firefox, Chrome, Safari, Opera
> * Mobile: iOS (Chrome, Opera Mini), Android (Chrome, Firefox)
> 
> All looks fine except one query re. hover state for the bottom-most link in Desktop Firefox - underline seems visually unclear:
> 
> Firefox

Is this just on Persian or other services too? This could be a separate issue if it's happening on multiple. Well spotted.

**Comment by skwong25** [comment]
Seems to be an existing issue across a few services with non-Roman scripts. Tested a sample below: 
Noticable: Hindi, Urdu, Persian  
Doesn't really affect:  Bengali, Chinese, Tigrinya, Korean, Gujarati, Sinhala, Arabic 

For some languages, the hover state underline is clear off the text but for others, it seems closer to the text which means that even if there are no downward protrusions, sections of underline are still missing? Result is a unclear hoverstate underline. 

See examples in screenshot: 

LIVE HINDI
LOCAL URDU

**Comment by pvaliani** [comment]
> Seems to be an existing issue across a few services with non-Roman scripts. Tested a sample below: Noticable: Hindi, Urdu, Persian Doesn't really affect: Bengali, Chinese, Tigrinya, Korean, Gujarati, Sinhala, Arabic
> 
> For some languages, the hover state underline is clear off the text but for others, it seems closer to the text which means that even if there are no downward protrusions, sections of underline are still missing? Result is a unclear hoverstate underline.
> 
> See examples in screenshot:
> 
> LIVE HINDI
> 
> LOCAL URDU

No worries. Just to clarify, a separate issue is being created for this hover-underline bug and I will proceed to merge the configuration updates. Thanks!

**Pull request merged**